docs/readmes/lte/build_install_magma_pkg_in_agw.md

```diff
@@ -5,71 +5,78 @@ hide_title: true
-    To build an AGW package, use the script located at `$MAGMA_ROOT/lte/gateway/fabfile.py`. The commands below will create a vagrant machine, then build and create a deb package.
+    To build an AGW package spin up a vagrant machine and then build and create a deb package.
 
-    The following commands are to be run from `$MAGMA_ROOT/lte/gateway` on your host machine.
-    To create a package for production, run
+    From `$MAGMA_ROOT/lte/gateway` on your host machine run:
 
     ```bash
-    fab release package
+    vagrant up magma
+    vagrant ssh magma
     ```
 
-    To create a package for development or testing, run
+    In the VM from `$MAGMA_ROOT` run:
 
     ```bash
-    fab dev package
+    bazel run //lte/gateway/release:release_build --config=production
     ```
 
-    The `dev` flag will compile all C++ services with `Debug` compiler flags and enable ASAN. This is recommended for testing only as it will impact performance. In contrast, the production package has C++ services built with `RelWithDebInfo` compiler flags.
-
-4. **Locate the packages**. Once the above command finished. You need to enter the VM to verify the deb packages are there.
+    To create a package for development or testing, run
 
     ```bash
-    vagrant ssh magma
-    cd ~/magma-packages/
+    bazel run //lte/gateway/release:release_build
     ```
 
-    You will need only the ones that say `magma_1.1.XXX` and `magma-sctpd_1.1.XXX` (for v1.1 versions)
+    Omitting the `--config=production` flag will compile all C++ services with `Debug` compiler flags and enable ASAN. This is recommended for testing only as it will impact performance. In contrast, the production package has C++ services built with `RelWithDebInfo` compiler flags.
 
-5. **Download the package**. You can download the files to your computer from the vagrant machine. To do so, you can install a vagrant plugin in your computer and then download the package from the VM to your computer with the following commands:
+4. **Locate the packages**.
+    Once the above command finished you can find the packages inside the VM:
 
     ```bash
-    vagrant plugin install vagrant-scp
-    vagrant scp magma: ~/magma-packages/<deb_package>
+    cd /tmp/packages
     ```
 
-6. **Upload the package to AGW** that you would like to install.
+    There should be two packages named `magma_1.9.XXX` and `magma-sctpd_1.9.XXX` (for v1.9 versions).
+
+5. **Copy the packages to the target machine**.
 
-7. **Install the package**. In order to install the new deb package in AGW, you can run
+6. **Install the package**.
+    In order to install the new deb package in AGW, you can run
 
     ```bash
     sudo apt -f install MAGMA_PACKAGE
     ```
 
-8. **Restart the magma services**
+7. **Restart the magma services**
 
     ```bash
     sudo service magma@* stop
     sudo service magma@magmad restart
     ```
 
-9. You can **verify the installed version** with
+8. You can **verify the installed version** with
```

Comment:
_Only FYI:_
Changing this is usually not necessary when markdown is correctly translated. In fact, Markdown numbered lists should usually only be numbered with `1.` for all items to avoid those changes and confusion.
E.g. a "randomly" numbered list and its rendering:
```markdown
1. First
1. Second
2. Third
43. Fourth
1. Fiveth
```
Results in:
1. First
1. Second
2. Third
43. Fourth
1. Fiveth